Conan.VisualStudio/Utils.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Threading;
+
+namespace Conan.VisualStudio
+{
+    public static class Utils
```

Comment:
may we use something more specific instead of just `utils`?

Reply:
Suggest a name, my plan is to gather common utilities like this run, maybe som print-to-log-and-output,... in a single place to make them more reusable.